x-pack/plugins/alerts/server/plugin.ts

```diff
@@ -301,6 +301,7 @@ export class AlertingPlugin {
   ): (request: KibanaRequest) => Services {
     return (request) => ({
       callCluster: elasticsearch.legacy.client.asScoped(request).callAsCurrentUser,
+      esClient: elasticsearch.client.asScoped(request).asCurrentUser,
```

Comment:
Just wondering if this line is needed if we have `scopedClusterClient: elasticsearch.client.asScoped(request).asCurrentUser,` line below?

Reply:
agree - I think that was just merged yesterday or so as well. Oh, funny, Matt's commit is older than JK's :-)

![image](https://user-images.githubusercontent.com/25117/96719916-11c41b00-1378-11eb-9ec6-af0609783ffc.png)